src/dealDashboard/dealClauses/dealClauses.scss

```diff
@@ -0,0 +1,94 @@
+@import "src/resources/elements/primeDesignSystem/variables";
+
+$hover-transition-speed-fast: 175ms;
+$hover-transition-speed-medium: 0.3s;
+$hover-transition-speed-slow: 0.4s;
+
+p {
```

Comment:
Can you please put all of this css into a single container class?

Reply:
updated. I used the component itself as the container. No need for an additional element